system/audit-logs/vendor/logstash-audit-external/templates/_logstash.conf.tpl

```diff
@@ -181,10 +181,19 @@ filter {
         {{ end -}}
       }
 
-      if "awx" in [cluster_host_id] {
+      if "awx" in [logger_name] {
         mutate {
           add_field => { "[sap][cc][audit][source]"  => "awx" }
-          remove_field => [ "event_data" ]
+          remove_field => [ "[event_data][artifact_data]" ]
+          remove_field => [ "[event_data][changed]" ]
+          remove_field => [ "[event_data][dark]" ]
+          remove_field => [ "[event_data][failures]" ]
+          remove_field => [ "[event_data][ignored]" ]
+          remove_field => [ "[event_data][ok]" ]
+          remove_field => [ "[event_data][processed]" ]
+          remove_field => [ "[event_data][res]" ]
+          remove_field => [ "[event_data][rescued]" ]
+          remove_field => [ "[event_data][skipped]" ]
```

Comment:
_remove_field_ takes an array as argument. So it would be possible to call it once for all fields.